src/Microsoft.IdentityModel.Protocols.OpenIdConnect/Json/OpenIdConnectConfigurationSerializer.cs

```diff
@@ -0,0 +1,615 @@
+                        JsonPrimitives.ReadStrings(ref reader, config.RequestObjectEncryptionAlgValuesSupported, MetadataName.RequestObjectEncryptionAlgValuesSupported, ClassName, true);
+
+                    else if (reader.ValueTextEquals(Utf8Bytes.RequestObjectEncryptionEncValuesSupported))
+                        JsonPrimitives.ReadStrings(ref reader, config.RequestObjectEncryptionEncValuesSupported, MetadataName.RequestObjectEncryptionEncValuesSupported, ClassName, true);
+
+                    else if (reader.ValueTextEquals(Utf8Bytes.RequestObjectSigningAlgValuesSupported))
+                        JsonPrimitives.ReadStrings(ref reader, config.RequestObjectSigningAlgValuesSupported, MetadataName.RequestObjectSigningAlgValuesSupported, ClassName, true);
+
+                    else if (reader.ValueTextEquals(Utf8Bytes.RequestParameterSupported))
+                        config.RequestParameterSupported = JsonPrimitives.ReadBoolean(ref reader, MetadataName.RequestParameterSupported, ClassName, true);
+
+                    else if (reader.ValueTextEquals(Utf8Bytes.RequestUriParameterSupported))
+                        config.RequestUriParameterSupported = JsonPrimitives.ReadBoolean(ref reader, MetadataName.RequestUriParameterSupported, ClassName, true);
+
+                    else if (reader.ValueTextEquals(Utf8Bytes.RequireRequestUriRegistration))
+                        config.RequireRequestUriRegistration = JsonPrimitives.ReadBoolean(ref reader, MetadataName.RequireRequestUriRegistration, ClassName, true);
+
+                    else if (reader.ValueTextEquals(Utf8Bytes.ResponseModesSupported))
+                        JsonPrimitives.ReadStrings(ref reader, config.ResponseModesSupported, MetadataName.ResponseModesSupported, ClassName, true);
+
+                    else if (reader.ValueTextEquals(Utf8Bytes.ResponseTypesSupported))
+                        JsonPrimitives.ReadStrings(ref reader, config.ResponseTypesSupported, MetadataName.ResponseTypesSupported, ClassName, true);
+
+                    else if (reader.ValueTextEquals(Utf8Bytes.ScopesSupported))
+                        JsonPrimitives.ReadStrings(ref reader, config.ScopesSupported, MetadataName.ScopesSupported, ClassName, true);
+
+                    else if (reader.ValueTextEquals(Utf8Bytes.ServiceDocumentation))
+                        config.ServiceDocumentation = JsonPrimitives.ReadString(ref reader, MetadataName.ScopesSupported, ClassName, true);
+
+                    else if (reader.ValueTextEquals(Utf8Bytes.SubjectTypesSupported))
+                        JsonPrimitives.ReadStrings(ref reader, config.SubjectTypesSupported, MetadataName.SubjectTypesSupported, ClassName, true);
+
+                    else if (reader.ValueTextEquals(Utf8Bytes.SubjectTypesSupported))
+                        JsonPrimitives.ReadStrings(ref reader, config.SubjectTypesSupported, MetadataName.SubjectTypesSupported, ClassName, true);
```

Comment:
are these two duplicates?

Reply:
@jennyf19 addressed in d789adbb30b8093a0eff70ee545983195935366f